class_Hand.py

```diff
@@ -0,0 +1,24 @@
+class Hand:
+    
+    def __init__(self, name):
+        self.cards = []
+        self.name = name
```

Comment:
Your hands only have two names "right" and "left". I'd better create a class `Player`, inherit it from `Hand` and add a name there. Does it make sense?